### PR TITLE
[#02]  CI 구성 

### LINKED ISSUE #2: CI ( github Action ) 초기 구축 작업
2023.04.11 
 - github Action에 대해  학습이 초기 구축 작업 시작

### PR DESCRIPTION
**Pull request by Whale0928**
## 작업 종류

- [ ]  버그 수정
- [ ]  기능
- [ ]  코드 스타일
- [ ]  리팩토링
- [ ]  빌드
- [X]  CI/CD
- [ ]  문서
- [ ] 기타

## 작업 내용

CI구성을 위한 PR입니다.
CI를 통해 기본적인 빌드 검증 후 
코드 컨벤션과 코드 커버리지를 체크합니다.

### CODE REVIEW

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package studyconnect.member.dto;
+
+
+import com.studyconnect.member.dto.LoginRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LoginRequestTest{
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	@Test
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	@DisplayName("LoginRequest 생성 테스트")
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	public void test() {
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+		LoginRequest loginRequest = new LoginRequest("test", "test1234");
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+		String memberEmail = loginRequest.getMemberEmail();
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+		Assertions.assertEquals("test", memberEmail);
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+	}
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/test/java/studyconnect/study/Study.java [hunk 9]
```diff
@@ -0,0 +1,4 @@
+package studyconnect.study;
+
+public class Study {
```
Comment (Whale0928) [inline src/test/java/studyconnect/study/Study.java:3]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 10]
```diff
@@ -0,0 +1,17 @@
+package studyconnect;
+
+import com.studyconnect.StudyConnectApplication;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Import statement for 'org.junit.jupiter.api.Assertions.*' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.

Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:8]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - org.junit.jupiter.api.Assertions.*.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+@SpringBootTest(classes = StudyConnectApplication.class)
+class StudyConnectApplicationTest {
+    @Test
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+    @DisplayName("Spring boot Start Test")
+    void contextLoads() {
+        System.out.println("Spring boot Start Test");
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+    }
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.

File: src/test/java/studyconnect/tag/Tag.java [hunk 14]
```diff
@@ -0,0 +1,4 @@
+package studyconnect.tag;
+
+public class Tag {
```
Comment (Whale0928) [inline src/test/java/studyconnect/tag/Tag.java:3]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java [hunk 15]
```diff
@@ -1,12 +1,20 @@
...
+ * LoginRequest Domain
  * The type LoginRequest.
  */
 @Getter
+@RequiredArgsConstructor
 public class LoginRequest {
 	private String memberEmail;
 	private String password;
+
+	public LoginRequest(String memberEmail, String password) {
```
Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java [hunk 16]
```diff
@@ -1,12 +1,20 @@
...
+		this.memberEmail = memberEmail;
```
Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java [hunk 17]
```diff
@@ -1,12 +1,20 @@
...
+		this.password = password;
```
Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java [hunk 18]
```diff
@@ -1,12 +1,20 @@
...
+	}
```
Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def rcurly' has incorrect indentation level 8, expected level should be 2.

**Comment by Whale0928** [comment]
checkStyle 예외 설정 수정 중입니다.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 19]
```diff
@@ -0,0 +1,17 @@
+package studyconnect.member.dto;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.studyconnect.member.dto.LoginRequest;
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.studyconnect.member.dto.LoginRequest' import. Should be before 'org.junit.jupiter.api.Test'.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+
+public class LoginRequestTest {
```
Comment (Whale0928) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 21]
```diff
@@ -0,0 +1,16 @@
+package studyconnect;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.studyconnect.StudyConnectApplication;
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.studyconnect.StudyConnectApplication'

Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.studyconnect.StudyConnectApplication' import. Should be before 'org.springframework.boot.test.context.SpringBootTest'.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 22]
```diff
@@ -0,0 +1,16 @@
...
+
+@SpringBootTest(classes = StudyConnectApplication.class)
+class StudyConnectApplicationTest {
+	@Test
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 23]
```diff
@@ -0,0 +1,16 @@
...
+	@DisplayName("Spring boot Start Test")
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/test/java/studyconnect/StudyConnectApplicationTest.java [hunk 24]
```diff
@@ -0,0 +1,16 @@
...
+	void contextLoads() {
```
Comment (Whale0928) [inline src/test/java/studyconnect/StudyConnectApplicationTest.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Comment by Whale0928** [comment]
체크 스타일 검증 옵션이 충돌이 많이 발생해 일단 제외했습니다 😭😭

File: .github/workflows/code-quality-check.yml
Comment (ifi999) [inline .github/workflows/code-quality-check.yml:1]:
```
jobs:
  coverage:
    runs-on: ubuntu-latest
```
이 부분에서 우분투 버전을 명시해두는 것이 좋지 않을까요?
gradle.yml에도 비슷한 부분이 있어요

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java
Comment (ifi999) [inline src/main/java/com/studyconnect/member/dto/LoginRequest.java:1]: `@RequiredArgsConstructor` 애너테이션과 아래 만든 생성자는 같은 역할인데 둘 중 하나만 남기는 것이 좋을 것 같아요

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java
Comment (ifi999) [inline src/test/java/studyconnect/member/dto/LoginRequestTest.java:1]: dto 생성자 테스트는 생각 못해봤었는데, 테스트 대상에 포함이 될까요? 궁금하네요

File: .github/workflows/code-quality-check.yml
Comment (Whale0928) [reply .github/workflows/code-quality-check.yml:1]: runs-on: ubuntu-latest 는 깃 허브에서 제공해주는 미리 구성된 환경이에요.
우분투의 최신 버전을 사용한다는 의미 인대 깃허브 에서 최적화한 버전 말고 별도의 버전을 명시하는 건 
해당 버전이 필요한 라이브러리나 이유가 있는 게 아니면 사용할 이유가 없다고 느껴졌어요

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java
Comment (Whale0928) [reply src/main/java/com/studyconnect/member/dto/LoginRequest.java:1]: 테스트 커버리지 생성 예시를 위해 만들어 진 거라 무의미한 코드입니다.
추후 해당 관련된 개발 하실 때 지우고 하시면 될 것 같아요.

File: src/test/java/studyconnect/member/dto/LoginRequestTest.java
Comment (Whale0928) [reply src/test/java/studyconnect/member/dto/LoginRequestTest.java:1]: 위와 동일한 이유에요 이벤트 동작 검증을 위해 더미로 만들어진 코드입니다.
추후 삭제하면 될 것 같아요

File: .github/workflows/code-quality-check.yml
Comment (ifi999) [reply .github/workflows/code-quality-check.yml:1]: 최신 버전을 사용한다는 의미인 것은 알고 있는데 라이브러리와의 호환성이 무조건 일치하는 건가요?
우분투 버전이 자동으로 올라갔을 때 `code-quality-check.yml` 을 사용하는 라이브러리와 무언가가 맞지 않아서 동작이 제대로 되지 않는다면 무엇이 문제인지 찾기 힘들지 않을까요

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java
Comment (ifi999) [reply src/main/java/com/studyconnect/member/dto/LoginRequest.java:1]: 잘 몰라서 궁금한 점인데 테스트 커버리지는 로컬 환경에서는 확인이 불가능한가요?
이 PR이 main에 올라갔을 때 Jacoco 보고서를 볼 수 있다는데 보고서 생성 확인을 위한 내용인가요?

File: .github/workflows/code-quality-check.yml
Comment (ifi999) [reply .github/workflows/code-quality-check.yml:1]: 제가 사용을 못해봐서 잘못 이해하고 있는 것일 수도 있다는 생각이 들었는데 참고하신 문서가 어떻게 되시나요?

Comment (Whale0928) [reply .github/workflows/code-quality-check.yml:1]: 음 아니에요 이건 근우님 말이 맞는 것 같아요  저 환경은 저희 서버 우분투 버전이랑 동일하게 관리 하는 게 좋을 것 같네요

File: src/main/java/com/studyconnect/member/dto/LoginRequest.java
Comment (Whale0928) [reply src/main/java/com/studyconnect/member/dto/LoginRequest.java:1]: 로컬에서도 동일하게 명령어로 리포트를 발급 시킬 수 있어요

Comment (ifi999) [reply src/main/java/com/studyconnect/member/dto/LoginRequest.java:1]: 아하 나중에 로컬에서 해봐야겠네요. 감사해요

**Review by ifi999 — APPROVED** [review]
CI 구성 확인했어요

**Pull request merged**